eslintrc.json

```diff
@@ -170,7 +170,8 @@
     "no-unused-vars": [2, {
       "vars": "all",
       "args": "none",
-      "ignoreRestSiblings": true
+      "ignoreRestSiblings": true,
+      "varsIgnorePattern": "^_|ignored$"
```

Comment:
are there scenarios for ignore patterns that `ignoreRestSiblings` does not cover?

Reply:
yeah, that project isn't on the latest version, I guess that's why it didn't work on that rule.  I still think this is a good change, though.  What about 

```js
try {
  bleh()
} catch (_) {
  return
}
```

Reply:
in that case just omit the `( )` 😄 

```js
try {
  bleh()
} catch {
  return
}
```

definitely not opposed to having some ignore patterns too though, I just had a hard time thinking of what cases actually exist for this when I added `ignoreRestSiblings` (which is why I went with that, instead of patterns)

also, as written this would match any variable that either starts with `_` or ends with `ignored` 

```js
// both are valid, do not cause lint errors
const foo_ignored = 'foo'
const _whatever = 'bar'
```

so maybe `"^(_|ignored)$"` is better?

ultimately I'm fine with the current version if other folks are, just throwing out some thoughts

Reply:
The current pattern is anchored, though, so it should be the same just without the parens. Am I missing something?

Reply:
If the `ignoreRestSiblings` does it, then I guess we don't need the change, but I'd still want to merge this PR with the test re-organization to read the fixtures from eslint.

Reply:
without the parens it's only anchored at one end or the other, not both

+1 to the test changes for sure

Reply:
ok, we'll go with just the test chnges.  My bad for not bumping on that other repo first :]